examples/backbone_marionette/css/app.css

```diff
@@ -7,3 +7,7 @@
 #footer {
 	display: none;
 }
+
+#clear-completed::after {
+	top: 0;
+}
```

Comment:
without the top rule it looks like it is being cleared to the bottom

![screen shot 2015-02-08 at 1 12 53 pm](https://cloud.githubusercontent.com/assets/883126/6097655/40aca49e-af94-11e4-8a17-971a0cb9c1d6.png)

quite odd really